frb/galaxies/prospector.py

```diff
@@ -0,0 +1,45 @@
+""" WARNING -- WIP -- Methods related to running and reading Prospector outputs """
+
+from frb.galaxies import nebular
+from frb.galaxies import utils as galaxy_utils
+from frb.galaxies import defs
+
+# TODO -- THIS NEEDS MORE FILTERS
+prospector_filter_dict = {}
+for key in defs.SDSS_bands:
+    prospector_filter_dict['SDSS_{:s}'.format(key)] = 'sdss_{:s}'.format(key)
+
+def load_spec_from_specdb(host_obj, INSTR:str, deredden=True,
```

Comment:
These functions could use more verbose docstrings with args and return values specified.